egs/fisher_callhome_spanish_st/st1/local/callhome_make_spk2gender.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
```

Comment:
If you want to use python2 here, rewrite this with `#!/usr/bin/env python2`. When user's python is python3, python3 should not be called here.

Reply:
@ShigekiKarita Sorry, these files are copies from kaldi repo. I didn't modify them.
@sw005320, is it OK to change these files?